ofl/mochiypopone/METADATA.pb

```diff
@@ -0,0 +1,19 @@
+name: "Mochiy Pop One"
+designer: "FONTDASU"
+license: "OFL"
+category: "HANDWRITING"
+date_added: "2020-12-07"
```

Comment:
They won't go live until 2020-12-14 so better to use that date

Reply:
Done